php/thankYou.php

```diff
@@ -438,6 +448,17 @@
         $mailArray[$i] = mysqli_real_escape_string($cnxn, $_POST["refEmail".$i]);
 
     }
+
+    $submited = saveVolunteer($cnxn, $firstName, $lastName, $email, $phoneNumber, $mailList, $tShirt, $street,
+                              $city, $state, $zip, $howDidHear, $motivation, $volExperience, $skills, $firstArray, $lastArray, $pNumberArray, $mailArray,
+                               $relationArray, $oneWeek, $weekend, $interestArray);
```

Comment:
Just noticed $weekend and $interestArray was never passed through the mysqli_real_escape_string(). We only need to be concerned about inserting the "Other" text input value for interests in the interest table though.